crawler/config/centre.py

```diff
@@ -29,6 +29,13 @@
 CENTRE_REGEX_BARCODE = r"^[\W_]*([\w-]*)_([A-Z]\d{0,1}\d)[\W_]*$"
 CENTRE_DIR_BACKUPS = "data/backups"
 CENTRE_REGEX_SFTP_FILE_HERON = r"sanger_report_(\d{6}_\d{4}).*\.csv$"
+AP_SURVEILLANCE_1 = r"^AP-rna-\d+\.csv$"
+AP_SURVEILLANCE_2 = r"^[a-zA-Z]{2}-[a-zA-Z]{3}-\d+\.csv$"
```

Comment:
`AP_SURVEILLANCE_1` is a subset of `AP_SURVEILLANCE_2` so you don't need the more specific one.